doc/source/programs/gdal_vector_geom_simplify.rst

```diff
@@ -0,0 +1,71 @@
+.. _gdal_vector_geom_simplify_subsubcommand:
+
+================================================================================
+"gdal vector geom simplify" sub-sub-command
+================================================================================
+
+.. versionadded:: 3.11
+
+.. only:: html
+
+    Simplify geometries of a vector dataset.
+
+.. Index:: gdal vector geom simplify
+
+Synopsis
+--------
+
+.. program-output:: gdal vector geom simplify --help-doc
+
+Description
+-----------
+
+:program:`gdal vector geom simplify` simplify geometries ensuring that no
+segment is longer than the specified tolerance.
+
+The algorithm used by :cpp:func:`OGRGeometry::SimplifyPreserveTopology`
+preserves the topology per feature, in particular for polygon geometries
+(eliminating interior rings that would be too small), but not for a whole layer.
+Thus gaps or overlaps between geometries that were initially contiguous may
+happen.
+
+This command can also be used as a step of :ref:`gdal_vector_pipeline_subcommand`.
+
+.. note:: This command requires a GDAL build against the GEOS library.
+
+Standard options
+++++++++++++++++
+
+.. include:: gdal_options/of_vector.rst
+
+.. include:: gdal_options/co_vector.rst
+
+.. include:: gdal_options/overwrite.rst
+
+.. include:: gdal_options/active_layer.rst
+
+.. include:: gdal_options/active_geometry.rst
+
+.. option:: --tolerance <TOLERANCE>
+
+    The specified value of this option is the tolerance used to merge
+    consecutive points of the output geometry using the
+    :cpp:func:`OGRGeometry::SimplifyPreserveTopology` method
+    The unit of the distance is in georeferenced units of the source layer.
+
+Advanced options
+++++++++++++++++
+
+.. include:: gdal_options/oo.rst
+
+.. include:: gdal_options/if.rst
+
+Examples
+--------
+
+.. example::
+   :title: Make sure that segments of geometries are not smaller than one km (assuming the CRS is in meters)
```

Comment:
Rather large tolerance for practical use, even OK in theory, and the description is wrong.

Reply:
what is wrong exactly ?

Reply:
the phrasing was indeed not totally accurate. I've reworded in a more neutral way: "Simplify geometries using a tolerance of one km (assuming the CRS is in meters)"

Reply:
Simplify does not do that. The simplified line can be however long, if no part of the simplified line is not moved more than by the tolerance from the original line.  Simplifying a straight line with lots of vertices will keep only the start and end points.